lib/src/components/ComponentWrapper.tsx

```diff
@@ -73,6 +74,8 @@ export class ComponentWrapper {
       }
     }
 
+    ReactLifecyclesCompat.polyfill(WrappedComponent);
```

Comment:
Is this the only way to support backwards compatibility? I would really like not having another dependency, and polyfilling seems like overkill..

Reply:
This is the way proposed by the react team. They talk a bit about it in this post https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html#open-source-project-maintainers.

The other thing that comes to my mind is to have two different WrappedComponent, one with the new lifecycle and another with the old one. Then we use the proper one based on the React version.

Reply:
👍